polkadot-parachains/integritee-runtime/src/xcm_config.rs

```diff
@@ -50,25 +52,31 @@ use xcm::latest::prelude::*;
 use xcm_builder::{
 	AccountId32Aliases, AllowKnownQueryResponses, AllowSubscriptionsFrom,
 	AllowTopLevelPaidExecutionFrom, CurrencyAdapter, EnsureXcmOrigin, FixedWeightBounds,
-	LocationInverter, ParentAsSuperuser, ParentIsPreset, RelayChainAsNative,
-	SiblingParachainAsNative, SiblingParachainConvertsVia, SignedAccountId32AsNative,
-	SignedToAccountId32, SovereignSignedViaLocation, TakeWeightCredit, UsingComponents,
+	ParentAsSuperuser, ParentIsPreset, RelayChainAsNative, SiblingParachainAsNative,
+	SiblingParachainConvertsVia, SignedAccountId32AsNative, SignedToAccountId32,
+	SovereignSignedViaLocation, TakeWeightCredit, UsingComponents,
 };
-use xcm_executor::{Config, XcmExecutor};
+use xcm_executor::XcmExecutor;
 use xcm_transactor_primitives::*;
 
+const fn teer_general_key() -> Junction {
+	const TEER_KEY: [u8; 32] = *b"TEER0000000000000000000000000000";
```

Comment:
@clangenb we found the cause of broken XCM ;-)
char '0' is hex 0x30, but should be 0x00

Reply:
omg, sorry for that.

Reply:
not blaming, just documenting ;-)